src/test/java/com/google/sps/DatastoreUserRepositoryTest.java

```diff
@@ -0,0 +1,101 @@
+package com.google.sps;
+
+import com.google.sps.user.User;
+import com.google.sps.user.repository.UserRepository;
+import com.google.sps.user.repository.UserRepositoryFactory;
+import com.google.sps.user.repository.impl.DatastoreUserRepository;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.sps.data.RepositoryType;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+
+@RunWith(JUnit4.class)
+public final class DatastoreUserRepositoryTest {
+    private static final String ID = "userid";
+    private static final String EMAIL = "user@gmail.com";
+    private static final String NAME = "username";
+    private static final String SELF_INTRODUCTION = "I am the user";
+    private static final String IMG_URL = "/img.com";
+
+    private static final String ID_B = "useridB";
+
+    private static final String ID_C = "useridC";
+    private static final String EMAIL_C = "user@gmail.com_C";
+    private static final String NAME_C = "username_C";
+    private static final String SELF_INTRODUCTION_C = "I am the user_C";
+    private static final String IMG_URL_C = "/img.com_C";
+
+    private final static User toSaveUser = new User.Builder(ID, EMAIL).setName(NAME).addSelfIntroduction(SELF_INTRODUCTION).addImgUrl(IMG_URL).build(); 
```

Comment:
longer than 100 chars. You can break the line as follow:

```suggestion
    private final static User toSaveUser = 
        new User.Builder(ID, EMAIL)
            .setName(NAME)
            .addSelfIntroduction(SELF_INTRODUCTION)
            .addImgUrl(IMG_URL)
            .build()
```

Reply:
Broke the line. 
I set up an IntelliJ setting which does this for me automatically, and used the format suggested by IntelliJ.